gui/src/Collaborators/components/projectMembers.tsx

```diff
@@ -43,13 +44,17 @@ export class ProjectMembers extends React.Component<Props & DispatchProps, any>
     this.props.fetchMembersForProject();
   }
 
+  onLoadLabelsClicked = (e: MouseEvent) => {
+    this.props.loadLabelsTab();
+  };
+
   render(): React.ReactElement<HTMLDivElement> {
     return (
       <div className="projectMembers">
         <p></p>
         <h1>Members</h1>
         <p className="lede">Theses are users who have access to this project, meaning they can see data, models and services associated with the project. Additionally, owners and collaborators can create new models, and new services based on those models.</p>
-        <p className="lede">Labels associated with projects have <span className="link" onClick={ this.props.loadLabelsTab }>their own access controls, shown here</span>.</p>
+        <p className="lede">Labels associated with projects have <span className="link" onClick={ this.onLoadLabelsClicked }>their own access controls, shown here</span>.</p>
```

Comment:
Sort of, it now enforces the arguments of the function, so a prop declared as just myFunction :Function can't be used by default in place of myFunction: Function(e: React.MouseEvent), but we're almost never using the MouseEvent data in the functions passed as props, so it'd be a pain to rewrite their actual signatures.

A more terse alternative would be onClick={ (e)=>this.props.loadLabelsTab }  or presumably in theory there's a way to do something like myFunction :Function<any> .